CHANGELOG.md

```diff
@@ -146,6 +146,7 @@
 - Update Content-Security-Policy to allow loading fonts from country configuration [#7296](https://github.com/opencrvs/opencrvs-core/pull/7296)
 - Fix frontend crashing on 'Registration by Status' under performance due to missing translations [#7129](https://github.com/opencrvs/opencrvs-core/pull/7129)
 - Fix email of practitioner to be saved in hearth. A migration is added to correct the email of practitoiner in existing db. [7315](https://github.com/opencrvs/opencrvs-core/pull/7315)
+- Fix EDIT not working 4 DateRangePickerForFormField [7485](https://github.com/opencrvs/opencrvs-core/pull/7485)
```

Comment:
This is still something we need to improve as a team but could we rephrase this a bit? If you imagine being a maintainer of an OpenCRVS country implementation trying to understand what has changed in the product and maybe not too familiar with the internals, what would you like it to read here? I'm thinking something along the lines of 
>Fix inaccessible and only partly visible "Edit" button in "Advanced Search" - feature's date range list